src/ServiceControl/App_Packages/StackTraceParser/StackTraceParser.cs

```diff
@@ -76,6 +78,7 @@ partial class StackTraceParser
             return Parse(text, (idx, len, txt) => txt,
                                (t, m) => new { Type = t, Method = m },
                                (pt, pn) => new KeyValuePair<string, string>(pt, pn),
+                               // ReSharper disable once PossibleMultipleEnumeration
```

Comment:
This should probably go because officially we no longer use Resharper. Otherwise it will trigger Adam analyzer tool

Reply:
Feedback from @janovesk: this is in an app_packages source file, so we can't change it.